### ISSUE TITLE
Is there a way of getting the id of the newly added record like you can do in ef?

### ISSUE THREAD

**Issue by sohnavcom** [magiccodingman/Magic.IndexedDb#9]

**Comment by sohnavcom** [comment]
I very  much like your product but like the guy in #8 always have these errors when retrieving data

**Comment by sohnavcom** [comment]
Any chance of a response???? Yours is the only wrapper that works on a Blazor Web App .Net 8 project??

**Comment by magiccodingman** [comment]
@sohnavcom Yes, give me a bit I'm working on it. I'm bouncing between a lot of projects right now. I'm going to try and get a lot of nice updates for everyone this week.

**Comment by magiccodingman** [comment]
Closing. Fixed / feature added by @yueyinqiu

**Comment by yueyinqiu** [comment]
@magiccodingman please take a lot at the current implemention:

https://github.com/magiccodingman/Magic.IndexedDb/blob/master/Magic.IndexedDb/IndexDbManager.cs#L131

`await CallJavascriptVoid(IndexedDbFunctions.ADD_ITEM, trans, RecordToSend);` is inside a nested `if` block, which means that it might not be invoked. Is is a mistake? Or the statements must be `true`?

**Comment by yueyinqiu** [comment]
Sorry but the link above will change with the new commits. And the below one is what I'm referring to previously:

https://github.com/magiccodingman/Magic.IndexedDb/blob/b689ce84336cdb13605124fcab8d39319e4aa46b/Magic.IndexedDb/IndexDbManager.cs#L134

When `convertedRecord == null` or `updatedRecord != null`, it won't call the js function. And the transaction won't be ended in this case?

**Comment by magiccodingman** [comment]
Resolved

**Comment by yueyinqiu** [comment]
Hmm... I think this is not resolved?
